Library/Sender.ts

```diff
@@ -156,6 +146,27 @@ class Sender {
                             this._storeToDisk(payload);
                         }
                     }
+                    // Redirect handling
+                    if (res.statusCode === 301 || // Moved Permanently
```

Comment:
Krishna, found with Application Insights that by default XHR will follow redirects (and there is no way to disable), for fetch the default is also to "follow" redirects (but you can change the default with the "redirect" option).

In both of the default cases you won't get the redirect status code.

For XHR (may be the same for fetch), he had to "review" the response URL to see if it was different it "infer" that a redirect occurred and so all new requests get redirected to the new endpoint.

Reply:
Node.js http native libraries will not handle redirects automatically so no need to worry about this scenario here